### DISCUSSION TITLE
Got nan value in output

### DISCUSSION THREAD

**Discussion by jtyang-chem** [deepmodeling/deepmd-kit#871] [Q&A]
I made all data by my scripts and give them into training, then I got a `lcurve.out` file with nan values.
- To make sure it's not a problem of version or input file, I also tested an example file of water box, everything goes well.
- my deepmd-kit version is v1.3.3 cpu.

I can't see the differences of which I made from the example data, I've already upload them to my [github](https://github.com/jtyang-chem/dpmd_nan), could someone help me?

**Comment by njzjz** [comment]
I can reproduce using the latest development version.

**Comment by njzjz** [comment]
You have atoms overlapping, which is the reason why you got nan values.

**Reply by jtyang-chem** [reply]
Thanks for the reply. Is it possible to add this situ to some error report?

**Reply by njzjz** [reply]
#914 will raise an error if some atoms are overlapping.
